topsky/TopSkyAreas.txt

```diff
@@ -1,7 +1,7 @@
-CATEGORYDEF:R:7:0:50:0:0:0:1:0:1
-CATEGORYDEF:D:7:0:50:0:0:0:1:0:1
-CATEGORYDEF:P:7:0:50:0:0:0:1:0:1
-CATEGORYDEF:T:7:0:50:0:0:0:1:0:1
+CATEGORYDEF:R:7:0:0:0:0:0:1:0:1
```

Comment:
TSA no longer filled, 1 cookie for who finds out the logic for being filled or not